src/types/composite/container.ts

```diff
@@ -4,9 +4,8 @@ import {isTypeOf} from "../basic";
 import {Type} from "../type";
 import {ContainerStructuralHandler, ContainerTreeHandler, ContainerByteArrayHandler} from "../../backings";
 
-export interface IContainerOptions {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  fields: Record<string, Type<any>>;
+export interface IContainerOptions<T extends Record<string, unknown>> {
```

Comment:
extends ObjectLike

Reply:
need to check. I think I can't use keyof with ObjectLike